crates/store/src/db/migrations/001-init.sql

```diff
@@ -23,16 +23,17 @@ CREATE TABLE
 CREATE TABLE
     notes
 (
-    block_num   INTEGER NOT NULL,
-    batch_index INTEGER NOT NULL, -- Index of batch in block, starting from 0
-    note_index  INTEGER NOT NULL, -- Index of note in batch, starting from 0
-    note_id     BLOB    NOT NULL,
-    note_type   INTEGER NOT NULL, -- 1-Public (0b01), 2-Private (0b10), 3-Encrypted (0b11)
-    sender      INTEGER NOT NULL,
-    tag         INTEGER NOT NULL,
-    aux         INTEGER NOT NULL,
-    merkle_path BLOB    NOT NULL,
-    details     BLOB,
+    block_num      INTEGER NOT NULL,
+    batch_index    INTEGER NOT NULL, -- Index of batch in block, starting from 0
+    note_index     INTEGER NOT NULL, -- Index of note in batch, starting from 0
+    note_id        BLOB    NOT NULL,
+    note_type      INTEGER NOT NULL, -- 1-Public (0b01), 2-Private (0b10), 3-Encrypted (0b11)
+    sender         INTEGER NOT NULL,
+    tag            INTEGER NOT NULL,
+    aux            INTEGER NOT NULL,
+    merkle_path    BLOB    NOT NULL,
+    details        BLOB,
+    execution_hint INTEGER,
```

Comment:
A couple of small comments:

- We should probably move `execution_hint` to be right after `aux` field since logically it is part of the note's metadata.
- I think `execution_hint` should be `NOT NULL` field as we always have an integer value for it.